### DISCUSSION TITLE
Ternoa Chain Era 103 - Slash

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/ternoa-hub#11] [General]
Hello. Here you can discuss the Era 103 slash topic. Feel free to share your thoughts and opinions about this here :)

Rules: Non-English comments will be deleted. Non related topic comments will be deleted.

**Comment by d4rk0snet** [comment]
On Era 103,

Node mars has been slashed because of duplicate session key on the network. The reason is a human error during another node build up.
More in details, the assumption was that it could be possible to cut the corner for BC synchronisation by duplicating bc storage folder to the other node. However, the node session key is also in this directory, that eventually lead to this situation.

This kind of error has been well identified and has not the result of intended harmful action towards the bc. It won't be reproduced and the slash should be removed if everybody agreed.

i always have been there for the community to help and to contribute, be assured that all i do is to help the ternoa to become stronger.

For now the slash is pending, the dead line is the 21 august.
